README-template.md

```diff
@@ -125,6 +125,10 @@ When Data Boost is enabled, the usage can be monitored by using Cloud Monitoring
 
 Dataproc [web interface]([url](https://cloud.google.com/dataproc/docs/concepts/accessing/cluster-web-interfaces)) can be used to debug especially to tune the performance. On the `YARN Application Timeline` page, it displays the execution timeline details for the executors and other functions. You can assign more workers if there are many tasks assigned to a same executor.
 
+### Root-partitionable Query
+
+When enable Data Boost, all queries passed to Cloud Spanner needs to be root-partitionable. Please see [`Read data in parallel`](https://cloud.google.com/spanner/docs/reads#read_data_in_parallel) for more details. If you see the issue when running Spark Spanner connector, probably the table you input is not supported. 
```

Comment:
Typos, please fix them to:

When DataBoost is enabled, all queries that are fed into Cloud Spanner must be root-partionable.... If you encounter an issue related to partitioning when using this connector, it is probably that the table being read from is not supported.